docs/source/configuration/persisted-queries.mdx

```diff
@@ -109,6 +109,27 @@ To enable safelisting, you _must_ turn off [automatic persisted queries](./in-me
 
 </Note>
 
+
+#### `local_safelist`
+
+Adding `local_safelist` to your `safelist` configuration allows you to use a local persisted query manifest in place of the hosted Uplink version. This is useful when you have an offline Enterprise license and are unable to use Uplink.
```

Comment:
In this implementation, there is no auto-reloading of the manifest file if it changes at runtime (I think?). Should this be called out explicitly in the docs as a limitation?

Reply:
yeah, good call. can add.